src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -49,6 +49,7 @@
         </xsd:choice>
 
         <xsd:attribute name="http-method-override" type="xsd:boolean" />
+        <xsd:attribute name="handle-all-throwables" type="xsd:boolean" />
```

Comment:
this change should not be part of the diff of this PR. Instead, we should first ensure that https://github.com/symfony/symfony/pull/51326 is merged up